src/mlpack/core/data/map_policies/missing_policy.hpp

```diff
@@ -0,0 +1,108 @@
+/**
+ * @file missing_policy.hpp
+ * @author Keon Kim
+ *
+ * Missing map policy for dataset info.
+ */
+#ifndef MLPACK_CORE_DATA_MAP_POLICIES_MISSING_POLICY_HPP
+#define MLPACK_CORE_DATA_MAP_POLICIES_MISSING_POLICY_HPP
+
+#include <mlpack/core.hpp>
+#include <unordered_map>
+#include <boost/bimap.hpp>
+#include <mlpack/core/data/map_policies/datatype.hpp>
+#include <limits>
+
+
+using namespace std;
+
+namespace mlpack {
+namespace data {
+
+/**
+ * Same as increment map policy, but does not change type of features.
+ */
+class MissingPolicy
+{
+ public:
+  // typedef of mapped_type
+  using mapped_type = double;
```

Comment:
@zoq it is because mappings can be any type of pairs. DatasetMapper takes care of mapping string values to any type of numeric values. 
For example, IncrementPolicy defines mapped_type as size_t. Therefore the DatasetMapper knows how to map and [return](https://github.com/mlpack/mlpack/pull/694/files#diff-4654fc03017f8f55e68d7e012a8c74f9R78) the data. (string to size_t mapping)
However, MissingPolicy uses NaN so the mapped_type is defined as double.